UnitedIncome/components
# Small doc improvement

### Description
> Please provide a brief description of your changes below.

Minor doc improvement that talks about the class prefixing. 

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

Read the change

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
